apps/coscrad-frontend/src/components/logout-button/logout-button.tsx

```diff
@@ -10,19 +10,21 @@ const LogoutButton = () => {
     const dispatch = useAppDispatch();
 
     return (
-        <IconButton
-            data-testid="logout-button"
-            color="secondary"
-            onClick={() => {
-                dispatch(userLoggedOut());
+        <Tooltip title="Logout">
```

Comment:
Depending on the size of the `Tooltip` text, something like "Click to log out" might make more sense. I'm not too opinionated, but we should probably develop a consistent standard for phrasing tooltips. It may be different in the context of a button as opposed to a form element. In the former case, you want to know what the button will do (like a PR commit message), in the latter, you want to know what the field is.

Reply:
Done